app/jobs/analytics_event_job.rb

```diff
@@ -0,0 +1,13 @@
+class AnalyticsEventJob < ActiveJob::Base
+  queue_as :analytics
+
+  TRACKER = if Rails.env.test?
+              Staccato.tracker(nil)
+            else
+              Staccato.tracker('UA-48605964-44', nil, ssl: true)
```

Comment:
Done in the second commit. This has the added benefit of simplifying the testing configuration.